include/bus_handler.hpp

```diff
@@ -116,6 +120,18 @@ class BusHandler
      * @param[in] event: Button event
      */
     void btnRequest(int event);
+
+    /**
+     * @brief API to execute panel function number.
+     *
+     * Method to trigger panel function via d-bus. Currently only certain
+     * functions are supported to trigger using d-bus calls.
+     *
+     * @param[in] funcNum - Function number to execute
+     * @return true if execution is triggered, false otherwise.
```

Comment:
Returns a tuple. Please mention that and also the value of fields in pass/fail condition. Like you have done in executor.hpp